### ISSUE TITLE
Analyzers don't have test coverage for VB, and codefixes are implemented only for C#

### ISSUE THREAD

**Issue by Youssef1313** [microsoft/testfx#4031]
## Summary

Currently, our analyzers are implemented both for C# and VB, but the tests are only validating C#.

And the codefixes are only implemented for C#, not implemented for VB at all.

**Comment by Evangelink** [comment]
Yes, that was a conscious call I made given the limited time we had for analyzers. I'll look if we have some data about numbers of VB users of MSTest to see how worth it it would be.

**Comment by Evangelink** [comment]
Closing as won't fix, let's see if we get request for it.
